test/test.js

```diff
@@ -403,6 +403,37 @@ describe("multi build", function(){
 		});
 	});
 
+	it("should pass the traceurOptions to transpile", function(done){
+		console.log('starting test');
+		rmdir(__dirname + "/es6-loose/bundle", function(error){
+			if(error) {
+				return done(error);
+			}
+
+			multiBuild({
+				config: __dirname + "/es6-loose/config.js",
+				main: "main",
+			}, {
+				quiet: true,
+				minify: false,
+				traceurOptions: {
```

Comment:
It's working because babelOptions is already on `opts` before it gets to https://github.com/stealjs/steal-tools/blob/minor/lib/graph/transpile.js#L53. That code isn't doing anything the way the test is written now.